labs/outer-loop-aks-azure.md

```diff
@@ -17,115 +17,124 @@ git branch --show-current
 
 ## Login to Azure
 
-- Login to Azure using `az login --use-device-code`
```

Comment:
we use bulleted lists throughout the document. Changing to normal text will remove consistency unless we change every doc which isn't feasible.

Reply:
FWIW, I did go through all the different labs and made them sentence-based instead of bullet lists in #24 but since I'm breaking these up into smaller PRs, it may seem more scattered. I'm still trying to get a few more smaller PRs in over  the next few days.